sdk/scala-sdk-protobuf/src/main/scala/kalix/scalasdk/impl/action/ActionAdapters.scala

```diff
@@ -156,4 +159,18 @@ private[scalasdk] final case class ScalaActionContextAdapter(javaSdkContext: jav
   }
 
   override def materializer(): Materializer = javaSdkContext.materializer()
+
+  override def componentCallMetadata: MetadataImpl = {
+    if (metadata.has(Telemetry.TRACE_PARENT_KEY)) {
+      new MetadataImpl(
+        new kalix.javasdk.impl.MetadataImpl(
+          List(
+            MetadataEntry(
+              Telemetry.TRACE_PARENT_KEY,
+              MetadataEntry.Value.StringValue(metadata.get(Telemetry.TRACE_PARENT_KEY).get)))))
```

Comment:
Better to do `metadata.get(...) match { case Some(value) => ... case None => }` because that can never go wrong.

Reply:
yup. 
Done.